code/modules/projectiles/projectile.dm

```diff
@@ -1026,6 +1025,15 @@ So if we are on the 32th absolute pixel coordinate we are on tile 1, but if we a
 	visible_message("<span class='danger'>[onlooker_feedback]</span>",
 	"<span class='xenodanger'>[victim_feedback]", null, 4)
 
+// Sundering procs
+/mob/living/proc/adjust_sunder(adjustment)
+	return 0
+
+/mob/living/proc/set_sunder(new_sunder)
+	return FALSE
```

Comment:
```suggestion
	return 0
```

Reply:
I originally planned for this to return FALSE because then calling the parent could override children

Reply:
The problem is returning a boolean in one proc and a numeric var in its redefinition. The type should be consistent, even if BYOND doesn't really distinguish between those two types.

Reply:
Well this is for set_sunder, i would expect this to only ever return true or false, based on if the parent handled it or not.

This should never return another value?

Reply:
Alright, that works.